### ISSUE TITLE
How to capture dashboard snapshot for a duration of test and embed it with test results

### ISSUE THREAD

**Issue by aather** [grafana/grafana#8427]
We use grafana 2.5.0 dashboards and graphite datasource in conjunction with abyss (https://github.com/aather/abyss) agents for performance analysis.  Considering we use public cloud and treat cloud instances as ephemeral, we therefore periodically clean up via cron associated metrics of terminated instances. We commonly use "share dashboard" and "share panel" or snapshot features of grafana to collaborate on problem resolution with other internal teams.  

We want to integrate grafana snapshot API into one of our benchmark project. Idea is to start metrics collection via abyss agents during benchmark. Once the benchmark is over, capture snapshot for duration of benchmark test by providing start and end time (Unix Timestamp) and embed URL with benchmark report, similar to:
http://xxxx/dashboard/db/system-performance?from=**1495209398000**&to=**1495213298000**&var-**Clusters=abyssagents**&var-Instance=**i-0a22f20c4ede65b1e**

Basically, I want to invoke snapshot API call with options: start and end time, cluster name and instance id (host name) and embed it into benchmark reports. Considering snapshot can last after instance is terminated and metrics are removed, this will allow us to keep snapshots with benchmarks results for an extended period of time.

I appreciate for any help you can provide.
Thanks,
Amer.

**Comment by torkelo** [comment]
please post questions to https://community.grafana.com/

**Comment by aather** [comment]
Sorry about that.
